README.md

```diff
@@ -53,7 +53,31 @@ $ cd my_project/
 $ ./bin/phpunit
 ```
 
+Docker
+------
+
+To run the "Symfony Demo Application" with Docker, [Download Docker][5] and [Docker Compose][6] and install them.
+
+At the root of the project, create an `.env.local` file and add the following line:
+
+```yaml
+DATABASE_URL=postgresql://127.0.0.1:5432/db?serverVersion=11&charset=utf8
+```
+
+Finally, run those commands:
+
+```bash
+$ cd my_project/
+$ docker-compose up -d
+$ symfony console doctrine:database:create
+$ symfony console doctrine:migrations:migrate
```

Comment:
I'd simplify with `doctrine:schema:create` instead, otherwise on each schema change we would need a new migration, but it's not necessary because we use SQLite here